homepage/tests/appUser_model_tests.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from homepage.models import AppUser
+
+
+@pytest.mark.django_db
+class TestAppUser:
+
+    @pytest.fixture
+    def app_users(self):
+        app_user1 = AppUser.create_app_user('user1', 'user1@mta.ac.il', 'useruser222')
+        app_user2 = AppUser.create_app_user('user2', 'user2@mta.ac.il', '1234')
+        app_user3 = AppUser.create_app_user('user3', 'user3@gmail.com', '10101010')
+
+        return (app_user1, app_user2, app_user3)
+
+    # The 3 appUsers created in the fixture above are:
+    # 1. The correct type of object (AppUser)
+    # 2. Successfully saved in the DB
+    def test_create_app_user(self, app_users):
+        for app_user in app_users:
+            assert isinstance(app_user, AppUser)
+            assert AppUser.objects.filter(pk=app_user.pk).exists()
+
+    # An app_user with invalid username is not created
+    def test_invald_username(self, app_users):
+        try:
+            AppUser.create_app_user('', 'user4@mta.ac.il', 'useruser222')
+        except ValueError:
+            invalid_username_created = False
+
+        assert invalid_username_created is False

```

Comment:
you can do it like:
```suggestion
        assert not invalid_username_created

```
but if it's better for you it is also ok :)

Reply:
i like it better with False